Longhelp.md

```diff
@@ -24,13 +24,16 @@ usage: marathon_lb.py [-h] [--longhelp] [--marathon MARATHON [MARATHON ...]]
                       [--command COMMAND] [--sse] [--health-check]
                       [--lru-cache-capacity LRU_CACHE_CAPACITY]
                       [--haproxy-map] [--dont-bind-http-https]
-                      [--ssl-certs SSL_CERTS] [--skip-validation] [--dry]
+                      [--ssl-certs SSL_CERTS]
+                      [--marathon-ca-cert MARATHON_CA_CERT]
```

Comment:
Verified that latest master and running it's pre-commit hook adds changes to Longhelp.md.